src/main/java/flab/delideli/service/MemberServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package flab.delideli.service;
+
+import flab.delideli.dao.MemberDao;
+import flab.delideli.dto.MemberDto;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class MemberServiceImpl implements MemberService{
```

Comment:
띄어쓰기도 맞춰주는게 좋습니다~ 추후 제안드릴거지만 `intellij checkstyle`에 대해 알아보시면 좋을 것 같아요

```suggestion
public class MemberServiceImpl implements MemberService {
```

Reply:
넵! 수정하겠습니다